src/SimulationRunner.cc

```diff
@@ -727,6 +770,9 @@ void SimulationRunner::Step(const UpdateInfo &_info)
   // Update all the systems.
   this->UpdateSystems();
 
+  // Update the step size and desired rtf
+  this->UpdatePhysicsParams();
```

Comment:
Yea we weren't sure in the first place where this would go. I thought about just moving it up to before systems are updated but realized it might result in incorrect UpdateInfo being sent over so I went for updating them as early as possible in the [Run loop](https://github.com/ignitionrobotics/ign-gazebo/pull/536/commits/6972fed90c1f351ffee1e81a7eb8fa5edaed9f69#diff-528565fd9b23fec7ea46e2f169d2098361634a77a74cac62166310c4624ca40dR693-R696).

Reply:
Ah interesting. The new location makes sense to me, even if we end up with the 2-step process. Thanks!